gnat2goto/driver/tree_walk.adb

```diff
@@ -830,6 +842,26 @@ package body Tree_Walk is
       return Args;
    end Do_Call_Parameters;
 
+   ------------------------
+   -- Do_If_Expression --
+   ------------------------
+
+   function Do_If_Expression (N : Node_Id) return Irep is
+      Expr : Node_Id := First (Expressions (N));
+      Cond : constant Irep := Do_Expression (Expr);
+      Then_Expr : Irep;
+      Else_Expr : Irep;
+      Expr_Type : constant Irep := Do_Type_Reference (Etype (N));
+   begin
+      Next (Expr);
+      Then_Expr := Do_Expression (Expr);
```

Comment:
Perhaps this is the best way, but I bet there are more specific accessors for the children of `N_If_Expression`

Reply:
It seems there are no specific accessors for the children. For some reason the condition, and if and else expressions are put into a list, but even if expressions with multiple `elsif` cases are parsed into nested if expressions with three operands each.